File: javascript/bottles.js
var bottle = function(bottles){
	if(bottles < 1){
		console.log("No more bottles of beer on the wall");
	}else{
	 console.log(`${bottles} bottles of beer on the wall. ${bottles} bottles of beer!!`);
	 console.log("take one down, pass it around.");
	 bottles -= 1
	 if(bottles != 0 ){	
	 	console.log(`${bottles} bottles of beer on the wall.\n`);
	 }
	 bottle(bottles);
	}
}
bottle(99);

File: python/bottles.py
def bottles(bottle):
	if bottle < 1:
		print "No more bottles of beer on the wall!"
	else:
		print "%s bottles of beer on the wall. %s bottles of beer!!"  %(bottle, bottle)
		print "take one down, pass it around."
		bottle -= 1
		if bottle != 0: 
			print "%s bottles of beer on the wall.\n" %(bottle)
		bottles(bottle)
	

bottles(99)
